components/Common/DarkModeToggle/index.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import DarkModeToggle from '.';
+import type { Meta, StoryObj } from '@storybook/react';
+import { ThemeProvider } from '../../../providers/themeprovider';
+
+const meta: Meta<typeof DarkModeToggle> = {
+  title: 'DarkModeToggle',
+  component: DarkModeToggle,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof DarkModeToggle>;
+
+export const Toggle: Story = {
```

Comment:
Is there a benefit on this approach over `https://github.com/nodejs/nodejs.org/blob/major/website-redesign/components/Header.stories.ts#L3`?

I suppose because our `app.tsx` doesn't has a `themeProvider` yet. Wouldn't it make sense to add the ThemeProvider here? https://github.com/nodejs/nodejs.org/blob/major/website-redesign/.storybook/preview.js#L26

Reply:
my bad will add it there 😅